README.md

```diff
@@ -74,9 +74,9 @@ PyCaret and its Machine Learning capabilities are seamlessly integrated with env
 
 ### Time Series
 
-PyCaret **new time series module** is now available in the `3.0-rc`. Staying true to the simplicity of PyCaret, it is consistent with the existing API and fully loaded with functionalities. Statistical testing, model training and selection (30+ algorithms), model analysis, automated hyperparameter tuning, experiment logging, deployment on cloud, and more. All of this with only a few lines of code. If you would like to give it a try, check out our official [quick start](https://nbviewer.org/github/pycaret/pycaret/blob/time\_series\_beta/time\_series\_101.ipynb) notebook.
+PyCaret **new time series module** is now available in the `3.0-rc`. Staying true to the simplicity of PyCaret, it is consistent with the existing API and fully loaded with functionalities. Statistical testing, model training and selection (30+ algorithms), model analysis, automated hyperparameter tuning, experiment logging, deployment on cloud, and more. All of this with only a few lines of code. If you would like to give it a try, check out our official [quick start](https://nbviewer.org/github/pycaret/pycaret/blob/master/examples/time_series/forecasting/time_series_101.ipynb) notebook.
```

Comment:
I Am wondering if we should change the link to the folder now that we have multiple files there: 

https://nbviewer.org/github/pycaret/pycaret/tree/master/examples/time_series/forecasting/

Reply:
That would be a better option, but in that case, I think that notebook names should be more informative.

Reply:
How about this in that case? It has more information about the notebooks.

https://pycaret.gitbook.io/docs/get-started/tutorials

Reply:
Yes, I think that would be fine.